pages/common/gitlab-runner.md

```diff
@@ -0,0 +1,28 @@
+# gitlab-runner
+
+> Run GitLab CI jobs.
```

Comment:
You're describing what a runner does, not the command itself. This sounds better for me:
```suggestion
> A CLI tool for managing your GitLab runners.
```
What do you think?